tests/WE2E/setup_WE2E_tests.sh

```diff
@@ -62,13 +62,13 @@ EXPTS_DIR=${TOP_DIR}/expt_dirs
 # Set the path to the machine-specific test suite file.
 #-----------------------------------------------------------------------
 
-auto_file=${scrfunc_dir}/machine_suites/${machine}.txt
+auto_file=${scrfunc_dir}/machine_suites/fundamental.txt
 
 #----------------------------------------------------------------------
 # Use exec_subdir consistent with the automated build.
 #----------------------------------------------------------------------
 
-exec_subdir='bin_intel/bin'
+exec_subdir='bin_intel/exec'
```

Comment:
The setup script uses `bin_intel/bin` before and i needed to change only the exec subdirectory for NCO. I could try and change both but seems to work fine with it except for the issue with NCO test case not being reported.

Reply:
I changed this to `install_intel/exec` since the directory `install_intel` is the installation directory and has `include`, `lib` and '`share` besides `exec`.